BusBoard.ConsoleApp/PostcodeObject.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Web.Hosting;
+
+namespace BusBoard.ConsoleApp
+{
+    public class PostcodeObject
+    {
+        public string Postcode { get; set; }
+        public string Longitude { get; set; }
```

Comment:
They're a string in the JSON and then just needed in a url so another string. So I thought it would be easier to keep them as a string rather than parse them twice?